internal/appsec/waf.go

```diff
@@ -22,6 +22,9 @@ import (
 	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
 )
 
+// samplerAppSec specifies that the span was sampled by AppSec.
+samplerAppSec int8 = 5
+
```

Comment:
Let's point at `ddtrace/tracer/sampler.go` here if we're not going to share the definitions. Otherwise I have no idea of the context of [`5`](https://en.wikipedia.org/wiki/42_(number)#The_Hitchhiker's_Guide_to_the_Galaxy).